src/package.json

```diff
@@ -7,7 +7,7 @@
     "@fortawesome/react-fontawesome": "^0.1.9",
     "react": "^16.10.2",
     "react-app-polyfill": "~0.2.2",
-    "react-bootstrap": "^1.0.0",
+    "react-bootstrap": "^1.4.0",
```

Comment:
Nice, glad you did this, thought not sure exactly what prompted the need for the change here.

Reply:
I wanted to switch an anchor-button to a `<Button>`, and when looking up how to do that I saw that the latest version supports `target` as a prop to `Button` and our version was throwing an error. Afterward, I merged your PR which looks like it uses `target`, so not sure how you didn't run into the error - the only difference I see is that you used `as='a'`.

Reply:
Yeah, I think I noticed that `target` previously didn't work unless you specified `as='a'`.